.gitignore

```diff
@@ -14,7 +14,7 @@ networktables.json
 
 ntcore/connectionlistenertest.json
 ntcore/timesynctest.json
-
+wpical/src/main/native/resources/camera calibration.json
```

Comment:
Why are we ignoring a file in the source tree?  This doesn't smell right.  Also, in general we avoid filenames with spaces in them.

Reply:
As the camera calibration json is created each time the tests are run, we figured it wouldn't make sense to have it saved to the repo. Should it be?

Reply:
Why is it get created in the src directory and not the build directory?

Reply:
The camera calibration saves the json to the same directory as the input video it was fed. We could make the save directory user configurable which would allow for saving the camera calibration to the build directory during the tests. Would that be better?

Reply:
Yes, that would be better.  It’s a devops/build system smell to generate files into the source tree.